recipes/sqlite3/all/conanfile.py

```diff
@@ -111,3 +111,6 @@ def package_info(self):
             bin_path = os.path.join(self.package_folder, "bin")
             self.output.info("Appending PATH env var with : {}".format(bin_path))
             self.env_info.PATH.append(bin_path)
+
+        self.cpp_info.names["cmake_find_package"] = "SQLite3"
```

Comment:
CMake target is `SQLite::SQLite3` https://cmake.org/cmake/help/v3.14/module/FindSQLite3.html

You could add a TODO comment.